public/app/features/dashboard-scene/saving/provisioned/SaveProvisionedDashboardForm.test.tsx

```diff
@@ -268,10 +276,10 @@ describe('SaveProvisionedDashboardForm', () => {
     const mockRequest = { ...mockRequestBase, isSuccess: true };
     (useCreateOrUpdateRepositoryFile as jest.Mock).mockReturnValue([mockAction, mockRequest]);
     const pathInput = screen.getByRole('textbox', { name: /path/i });
+    expect(pathInput.getAttribute('readonly')).toBe('true'); // can not edit the path value
```

Comment:
@Clarity-89 -- this does not work -- is there a way to check that this is readonly?

Reply:
I think it'd something like toHaveAttribute('readonly').